README.md

```diff
@@ -41,7 +41,13 @@ better queries which in turn will produce better results.
 To start developing open a terminal and run:
 
 ```
-$ nix-shell --run "yarn dev"
+$ cd src
```

Comment:
```suggestion
`nix-shell --run "cd frontend && yarn dev"`
```

The frontend was moved to `frontend/` folder. I would remove the rest.
Also there is no need to install yarn dependencies since they are installed with nix and a `frontend/node_modules` symlink is created.